package.json

```diff
@@ -22,6 +22,9 @@
   ],
   "author": "Minh Nguyễn",
   "license": "BSD",
+  "dependencies": {
+      "http-server": ">=14.1.1"
```

Comment:
Should this go under `devDependencies`? This package is used in some Web applications that wouldn’t be able to spin up an HTTP server.

Reply:
Correct. For context, a Web application, such as OpenStreetMap’s iD editor, uses NPM to fetch and build its dependencies, including this package and its transitive dependencies. The entire built codebase including dependencies winds up as JavaScript running in a Web browser.

Anything that’s only needed for updating the JSON files in this repository should go in `devDependencies`, because normally updating the files is something only a maintainer (or the Toolforge service) would ever need to do. Everyone else would simply consume the JSON files as they are.

Reply:
Instead of requiring the http server from npm, I used the builtin one from python, the only difference right now is that it allows to list directories (that before it was not allowed), not sure if that's something you don't want though.

Let me know :)